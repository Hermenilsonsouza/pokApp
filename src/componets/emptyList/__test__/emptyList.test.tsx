import React from 'react';
import {render} from '@testing-library/react-native';
import EmptyList from '../emptyList';

describe('<EmptyList />', () => {
  const components = () => render(<EmptyList />);

  it('should render <EmptyList />', () => {
    const rendered = components();
    expect(rendered).toMatchSnapshot();
  });
});
