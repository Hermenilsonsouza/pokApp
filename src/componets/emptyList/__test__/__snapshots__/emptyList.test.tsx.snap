// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EmptyList /> should render <EmptyList /> 1`] = `
<Image
  source={
    Object {
      "testUri": "../../../src/assets/images/error.png",
    }
  }
  style={
    Object {
      "height": 666.6666666666666,
      "width": 750,
    }
  }
/>
`;
